.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: dart
 
 dart:
+- "2.3.0"
```

Comment:
Better use `stable` here instead of hardcoded version.

Reply:
`stable` is bad if `stable` is different to your specified minimum supported SDK, which it is right now as stable is 2.3.1. By explicitly setting travis to test the explicit minimum supported SDK you won't accidentally introduce unsupported features, e.g. set literals, ui as code features, and in the future, not null by default type annotations.